README.md

```diff
@@ -1,45 +1,130 @@
+
+```
+import 'pdap-design-system'
+```
+
+And voila! You have access to PDAP's design system CSS / images to use in your templates / components:
+
+```
+<button class="button">Say Hello</button>
+```
+
+```
+<img src=node_modules/pdap-design-system/dist/images/{name-of-image}.{png | svg | gif} alt="some descriptive alt text" />
+```
+
+2. If the project does not have a build tool that bundles CSS for you, you can still include PDAP styles via HTML `link` tags:
+
+```
+  <link href="node_modules/pdap-design-system/dist/css/global-styles.css" rel="stylesheet" type="text/css">
+```
+
+### To import and use images from the package
+
+There are also two strategies for including PDAP image assets.
+
+1. If the project bundles images via a build tool like Webpack or Vite, you can import the package directly in your entrypoint js file. So, if your entrypoint is `main.js`, import it in that file like so:
+
+2. You can then reference images directly from the `node_modules` directory in the same way CSS is imported  
+   _note: If using a bundled framework like Vue.js, images must be imported into the bundle either individually or by importing this entire package, otherwise your app will not know where to look for the path you pass to src_
+
+```
+  <img src='node_modules/pdap-design-system/dist/images/{name-of-image}.{png | svg | gif}' alt='some descriptive alt text'>
+```
+
+### Using the CLI to copy assets to your local project directory
+
+If you want either styles or images copied to a local folder, you can do so from the root directory of your project.
+
+Assets can be copied using the `pdap-design-system` command line method exposed by this package.
+
+One of the following arguments is required:  
+`--copy-assets`: Copies all images and styles.
+`--copy-images`: Copies all images.
+`--copy-styles`: Copies all styles.
+
+The following argument is optional:  
+`--to={path}`: Path to directory where assets should be copied. Defaults to `assets`
+
+```
+  pdap-design-system --copy-images --to=image-assets
+```
+
+## Development Setup
+
+1. Clone the repo
+
+```
+gh repo clone Police-Data-Accessibility-Project/design-system
+```
+
+2. CD into the project folder and install dependencies
 
 ```
-  <link href="node_modules/pdap-design-system/system/css/normalize.css" rel="stylesheet" type="text/css">
-  <link href="node_modules/pdap-design-system/system/css/global-styles.css" rel="stylesheet" type="text/css">
-  ```
+cd design-system
+npm i
+```
+
+3.
 
+- To build styles to the `dist` folder:
 
+```
+npm run styles
+```
+
+- To watch for changes and update css as you make changes:
+
+```
+npm run styles:watch
+```
+
+4. If you use VS Code as your editor, you may want to install the [tailwind VS Code extension](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss), which helps with intellisense and the custom at-rules used by TailwindCSS.
 
 ## Principles
```

Comment:
now that we're using Tailwind, I think it'd be helpful to add some tips here for people (i.e. me) who would like to make changes to how things look without messing up / sidestepping the tailwind implementation.

- how can i be sure i'm not messing up other projects as I make changes? (version control saves us from this, but also reference the steps people can use to test the local symlink rather than publishing and importing a new package when you want to test) 
- how should i think about making changes to visual appearance? when should i use local styles vs making changes to global styles? has tailwind published a guide with tips for this we can use?

Reply:
> For my part of the review I will need some time to talk this through, ask basic questions, and make sure I understand. Maybe we could all talk sometime soon just to get on the same page, as this is so foundational.

@josh-chamberlain Absolutely, I think that's important here. Let's chat through it tomorrow.

> Based on "note any differences in styling" it seems like you're expecting there to be differences in appearance, right?

Yes, there were several things that were inconsistent or that I made opinionated decisions about because the tailwind architecture forced a change (the dark mode stuff for one thing). So I guess I'm asking where things are brand-inconsistent, rather than inconsistent with the prior demo.

> how can i be sure i'm not messing up other projects as I make changes? (version control saves us from this, but also reference the steps people can use to test the local symlink rather than publishing and importing a new package when you want to test)

💯 I'll update the `README` tomorrow. 
To that point, we probably want to protect `npm publish` an access token or (even better) implement a workflow in GH actions to publish automatically on merges to `main` when certain conditions are met, and gate `npm publish` behind a token so it can't be used outside of GH. Otherwise, anybody with access to the repo can just publish a new version whenever.

> how should i think about making changes to visual appearance? when should i use local styles vs making changes to global styles? has tailwind published a guide with tips for this we can use?

Honestly, the answer is to just use Tailwind's utility classes. But when Tailwind compiles CSS, it only compiles the classes that are actually used in the project. So for any consuming application that isn't using Tailwind, I think we could use the rule of 3s. If we repeat a thing 3 times, we create a utility class in `design-system` that abstracts it, and then publish a new version. But this is something that we probably want to discuss high-level guidelines for as well, so let's chat!

Reply:
<img src="https://github.com/Police-Data-Accessibility-Project/design-system/assets/30379833/0e21c5f9-8f4d-403c-870e-912e822714ac" width="60%" />
<img src="https://github.com/Police-Data-Accessibility-Project/design-system/assets/30379833/f5b1d307-b86f-4048-a0fa-85147492f2b8" width="60%" />

To be clear, I mean these visual differences ↑

There were enough small differences that I figured I might not be setting things up right / understanding how this works. Are these cases of our custom styles losing a fight with tailwind's defaults? Do we just need to do more customization, and this is expected?

I'm down to follow [their recommendations](https://tailwindcss.com/docs/adding-custom-styles) about custom styles, i.e. do it almost never. I don't mind the repetitiveness of tailwind is by design, right?

Reply:
> To be clear, I mean these visual differences 

@josh-chamberlain Yikes, that's my fault! I think I forgot to check the light mode styles after I re-worked the dark mode stuff. I'll go back and update this stuff to be more consistent.

But yes, re: your other point: I think if we're using TW we should align with how it's meant to be used (the one downside being that it's pretty repetitive).